.gitignore

```diff
@@ -45,3 +45,5 @@ autowiring.kdev4
 autowiring.VC.db
 .vscode
 AutowiringVersion.h
+
+*.VC.db*
```

Comment:
`Autowiring.VC.db is already listed above though?

Or if the intent here is the trailing asterisk, are there circumstances when it makes a db1, db2 file?